version.txt

```diff
@@ -1 +1 @@
-0.6.12
+0.6.13
```

Comment:
Shouldn't that be a breaking change?

Reply:
What do you see it breaking? It doesn't modify our CRDs.

Reply:
If someone is building their own discovery handler in some repo, doesn't this change require them to update their code as well? Or is the assumption that others doing so will be locked to a specific version?

Reply:
If the discovery handler is running in its own pod (not a part of Akri's repository), then it can run whatever versions it wants.